owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/settings/SettingsPictureUploadsViewModel.kt

```diff
@@ -84,10 +84,26 @@ class SettingsPictureUploadsViewModel(
     }
 
     fun useWifiOnly(wifiOnly: Boolean) {
+        viewModelScope.launch(coroutinesDispatcherProvider.io) {
+            savePictureUploadsConfigurationUseCase.execute(
+                if (!wifiOnly) {
+                    SavePictureUploadsConfigurationUseCase.Params(
+                        composePictureUploadsConfiguration(wifiOnly = wifiOnly, chargingOnly = false)
+                    )
+                } else {
+                    SavePictureUploadsConfigurationUseCase.Params(
+                        composePictureUploadsConfiguration(wifiOnly = wifiOnly)
+                    )
+                }
```

Comment:
```suggestion
                    SavePictureUploadsConfigurationUseCase.Params(
                        composePictureUploadsConfiguration(wifiOnly = wifiOnly, chargingOnly = false)
                    )
```

If we disable wifiOnly, chargingOnly should be disabled too. That's ok, but if we enable it, it means that chargingOnly was already disabled, so, we don't need this `if else`, right?

Anyway, I would try to use 

`if (condition) { } else { }` instead of `if(!condition) { } else { }`